.github/workflows/tests-integration.yml

```diff
@@ -63,6 +63,14 @@ jobs:
         if [ ${{ matrix.python-version }} = 3.6 ]; then
           pytest tests/integration_tests \
             --ignore tests/integration_tests/test_botorch.py
+        elif  [ ${{ matrix.python-version }} = 3.9 ]; then
+          pytest tests/integration_tests \
+            --ignore tests/integration_tests/test_keras.py \
+            --ignore tests/integration_tests/test_tensorflow.py \
+            --ignore tests/integration_tests/test_tfkeras.py \
+            --ignore tests/integration_tests/test_sklearn.py \
+            --ignore tests/integration_tests/test_tensorboard.py \
+            --ignore tests/integration_tests/allennlp_tests 
```

Comment:
Does allennlp support python 3.9? The test is supposed to be fine locally.

Reply:
Thanks for the review @nzw0301. Made the suggested changes in the latest commit.